docs/home/howto/OS_PATCHING.md

```diff
@@ -100,3 +115,51 @@ Available options:
 ## Additional information
 
 Red Hat provides notifications about security flaws that affect its products in the form of security advisories. For more information, see [here](https://access.redhat.com/security/updates/advisory).
+
+### Ubuntu
+
+For automated security patches Ubuntu uses unattended-upgrade facility. By default it runs every day. To verify it on your system, execute:  
+`dpkg --list unattended-upgrades`
+`cat /etc/apt/apt.conf.d/20auto-upgrades | grep Unattended-Upgrade`
+
+For information how to change Unattended-Upgrade configuration, see [here](https://github.com/mvo5/unattended-upgrades/blob/master/README.md).
+
+The following steps will allow you to perform an upgrade manually.  
+
+Update your local repository cache:  
+`sudo apt update`
+
+Verify if epirepo is disabled and official repositories are enabled:  
+`apt-cache policy`
+
+Verify your current Linux kernel version:  
+`uname -r`
```

Comment:
Same as above ... why do I verify this? What are desired outputs? How does it influence my upgrade process?

Reply:
It's not that important.. It's rather **good to know** your current kernel version before an upgrade.

Reply:
If that does not influence procedure, I would not add it as it just raises a questions. I think that if you want to keep it, maybe add separate section in `#patching-with-package-manager` section with some "pre-upgrade facts collecting".

Reply:
It's ok for me to remove it.